app/views/user/show.html.erb

```diff
@@ -127,7 +133,10 @@
 <% if @show_batches %>
   <% if @is_you && !@display_user.info_request_batches.empty? %>
     <h2 class="batch_results" id="batch_requests">
-      <%= n_('Your {{count}} batch requests', 'Your {{count}} batch requests', @display_user.info_request_batches.size, :count => @display_user.info_request_batches.size) %>
+      <%= n_('Your {{count}} batch requests',
+             'Your {{count}} batch requests',
```

Comment:
I know this is just a formatting change, but shouldn't the first of these be `'Your {{count}} batch request'`?